packaging/RPMS/Fedora/rabbitmq-server.spec

```diff
@@ -11,6 +11,7 @@ Source2: rabbitmq-script-wrapper
 Source3: rabbitmq-server.logrotate
 Source4: rabbitmq-server.ocf
 Source5: README
+Source6: rabbitmq-server-ha.ocf
```

Comment:
Do you mean inserting ocf as source5 and renaming all shifted source5 to source6 ? This would introduce more unneeded changes, that is the point?